packages/providers/upload-aws-s3/src/index.ts

```diff
@@ -107,19 +117,20 @@ module.exports = {
           );
         });
       },
-      uploadStream(file, customParams = {}) {
+      // uploadStream(file: File, customParams = {}) {
```

Comment:
Do we want to keep this?

Reply:
We should keep it.
`packages/core/upload/server/services/provider.js`

If uploadStream is not implemented in the provider, the asset stream is converted into a buffer. So effectively we would be loading the entire assets into memory .